source/tutorial/hbs-helper.md

```diff
@@ -0,0 +1,81 @@
+So far, our app is directly showing the user data from our Ember Data models.
+As our app grows, we will want to manipulate data further before presenting it to our users.
+For this reason, Ember offers handlebars template helpers to decorate the data in our templates.
+Let's use a handlebars helper to allow our users to quickly see if a property is "standalone" or part of a "Community".
+
+To get started, let's generate a helper for `rental-property-type`, note that helpers must have at least one dash,
+so we could not call our helper `type`:
+
+```shell
+ember g helper rental-property-type
+```
+
+This will create two files, our helper and it's related test:
+
+```shell
+installing helper
+  create app/helpers/rental-property-type.js
+installing helper-test
+  create tests/unit/helpers/rental-property-type-test.js
+```
+
+Our new helper starts out with some boilerplate code from the generator:
+
+```app/helpers/rental-property-type.js
+import Ember from 'ember';
+
+export function rentalGroup(params/*, hash*/) {
+  return params;
+}
+
+export default Ember.Helper.helper(rentalGroup);
+```
+
+To use this helper, we can use the helper's name within a handlebars block and pass a set of arguments.
```

Comment:
I think you could even strike this line, as the example below makes everything clear.